theme/js/components/reuse/card.vue

```diff
@@ -2,43 +2,46 @@
 Vue and simpler -->
 
 <template>
-  <article class="reuse-card">
+  <article class="reuse-card text-align-center fr-text--sm fr-mb-0">
     <Placeholder
-      class="reuse-image mb-sm"
-      :alt="title"
-      backgroundImage
+      class="reuse-image fr-mb-3v fr-responsive-img"
```

Comment:
I think the ratio on suggest can still be squeezed like this (on a 900*750 screen):
![image](https://user-images.githubusercontent.com/28541902/151128183-ad210f1a-649a-406a-899e-33bb8cd7cc6a.png)

Reply:
I removed some padding on suggest to improve this in f288583.